geode-core/src/main/java/org/apache/geode/internal/cache/TXCommitMessage.java

```diff
@@ -923,6 +923,11 @@ private boolean hasFlagsField(final DataOutput out) {
     return v.compareTo(Version.GEODE_180) >= 0;
   }
 
+  private boolean hasFlagsField(final DataInput out) {
```

Comment:
rename "out" to "in". :-)

Reply:
Saw that, already have a new commit ready. Having some failures in some other tests though.